tor/__init__.py

```diff
@@ -1,49 +1 @@
-import bugsnag
-from addict import Dict
-
 __version__ = '2.6.14'
```

Comment:
We don't have a good system for incrementing versions. My system loosely follows semantic versioning, but only when I remember to update the version number. Then I usually bump it one or two extra to make up for the releases I missed. It's very unofficial.